secrets/secrets-kubernetes/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM golang:1.22 AS builder
+ARG TARGETOS
+ARG TARGETARCH
+
+WORKDIR /workspace
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+
+COPY *.go .
+COPY pkg/ pkg/
+
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH:-amd64} go build -a -o secrets-kubernetes
```

Maybe worth moving the defaults up to the [`ARG` section](https://docs.docker.com/reference/dockerfile/#arg)?
```suggestion
RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -a -o secrets-kubernetes
```

---

done!